### PR TITLE
32 add set count function into interop

### PR DESCRIPTION
**Pull request by agehrin50**

### CODE REVIEW

File: BoxwriterResmarkInterop/Handlers/SetCountCommandHandler.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    public SetCountCommandHandler(IOPCUAService opcuaService)
+    {
+        _opcuaService = opcuaService;
+    }
+
+    public async Task<StringResponse> Handle(SetCountRequest request, CancellationToken cancellationToken)
+    {
+        var printerId = request.Data.ExtractPrinterId();
+        UInt64.TryParse(request.Data.ExtractAdditionalParameter(), out var startCount);
```
Comment (agehrin50) [inline BoxwriterResmarkInterop/Handlers/SetCountCommandHandler.cs:29]: This was a pain to figure out. The firmware needs this parameter to be Uint64 or the call fails.

Comment (erikgabbard) [reply BoxwriterResmarkInterop/Handlers/SetCountCommandHandler.cs:29]: Can you not just cast the return from `request.Data.ExtractAdditionalParameter()` like this:
`var startCount = (UInt64)request.Data.ExtractAdditionalParameter();`

**Review by erikgabbard — CHANGES_REQUESTED** [review]

File: BoxwriterResmarkInterop/Handlers/SetCountCommandHandler.cs (hunk 1 above)
Comment (agehrin50) [reply BoxwriterResmarkInterop/Handlers/SetCountCommandHandler.cs:29]: image

Comment (erikgabbard) [reply BoxwriterResmarkInterop/Handlers/SetCountCommandHandler.cs:29]: Maybe just `Convert.ToUInt64(value)`

Comment (agehrin50) [reply BoxwriterResmarkInterop/Handlers/SetCountCommandHandler.cs]: Yep that worked

**Review by erikgabbard — APPROVED** [review]

**Review by Gishe — CHANGES_REQUESTED** [review]
Please add a few unit tests against this call.

**Review by Drise13 — APPROVED** [review]

**Review by Gishe — APPROVED** [review]

**Pull request merged**